backend/app/models/experiment_template.py

```diff
@@ -34,19 +41,75 @@ class ExperimentTemplate(Document):
     updated_at: datetime = Field(default_factory=partial(datetime.now, tz=timezone.utc))
     state: TemplateState = TemplateState.CREATED
     retry_count: int = 0
-    approved: bool = False
+    is_approved: bool = False
```

Comment:
I think it would be better to stick with simple standard terms `approved`, `public` and `archived`.